be/src/column/json_column.cpp

```diff
@@ -85,4 +85,11 @@ uint32_t JsonColumn::serialize(size_t idx, uint8_t* pos) {
     return static_cast<uint32_t>(get_object(idx)->serialize(pos));
 }
 
+void JsonColumn::serialize_batch(uint8_t* dst, Buffer<uint32_t>& slice_sizes, size_t chunk_size,
+                                 uint32_t max_one_row_size) {
+    for (size_t i = 0; i < chunk_size; ++i) {
+        slice_sizes[i] += serialize(i, dst + i * max_one_row_size + slice_sizes[i]);
```

Comment:
That each row occupies max_one_row_size bytes wastes a lot memory, so suggest to  sequeeze the unused bytes away.

Reply:
yeah! you are right, it's will cost so many memory, but it's need modify the aggregate framework, I can modify it in other PR?

Reply:
need add a TODO